input/resources/StructureDefinition-indv-measurereport-deqm.xml

```diff
@@ -119,27 +119,23 @@
       <max value="*"/>
       <type>
         <code value="Extension"/>
-        <profile value="http://hl7.org/fhir/StructureDefinition/measurereport-supplementalData"/>
+        <profile value="http://hl7.org/fhir/5.0/StructureDefinition/extension-MeasureReport.supplementalData"/>
       </type>
       <mustSupport value="true"/>
     </element>
     <element id="MeasureReport.extension:supplementalData.extension:criteriaReference">
       <path value="MeasureReport.extension.extension"/>
       <sliceName value="criteriaReference"/>
-      <min value="0"/>
-      <max value="1"/>
```

Comment:
FHIR-46864 did not want to remove the criteriaReference or the populationDescription extensions. Removing lines 129 and 130 causes the extension to be removed. See the [differential table](https://build.fhir.org/ig/HL7/davinci-deqm/branches/FHIR-46864-link-goes-to-deprecated-extension-2-25/StructureDefinition-indv-measurereport-deqm.html).

However, a new ticket, FHIR-50091, needs both the criteriaReference and the description extensions to be moved to the value[x] element of the supplementalData extension. That new ticket could be included with this pull request.